.cci.jenkinsfile

```diff
@@ -15,8 +15,36 @@ parallel rpms: {
         git fetch origin --tags
         git submodule update --init
 
+        env SKIP_INSTALLDEPS=1 ./ci/build.sh
+      """
+      try {
+        sh """
+        make check
```

Comment:
I'd put this in a separate step, e.g. `Check` or something.
When tests fail, having more distinct and descriptive stages makes it easier to narrow down what went wrong I find. But yeah, totally good to just tweak this later too!